pom.xml

```diff
@@ -17,10 +17,11 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-        <camel.version>4.9.0</camel.version>
-        <citrus.version>4.5.1</citrus.version>
+        <camel.version>4.11.0</camel.version>
+        <camel.quarkus.platform.version>3.22.0</camel.quarkus.platform.version>
+        <citrus.version>4.6.0</citrus.version>
         <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
-        <quarkus.platform.version>3.17.0</quarkus.platform.version>
+        <quarkus.platform.version>3.22.1</quarkus.platform.version>
```

Comment:
also, 3.22.3 ?

or is there a diff  (not sure why the two 3.22 things are not aligned)